pages/contact.html

```diff
@@ -134,10 +134,12 @@ <h4 class="ui inverted header">Mount Sion C.B.S Waterford</h4>
                 </div>
             </div>
         </div>
+        <div class="ui section divider">
```

Comment:
just a small thing, if you can move the closing `</div>` so that this looks like: `<div class="ui section divider"></div>` we do not want to wrap the list of social media links in a divider, we just want the divider to separate the two bodies of text